assets/aml-benchmark/components/src/aml_benchmark/utils/io.py

```diff
@@ -20,6 +20,10 @@
 
 logger = get_logger(__name__)
 
+DATASTORE_DIRECTORY_URL_TEMPLATE = "AmlDatastore://{datastore_name}/{directory_name}"
+RANDOM_IMAGE_DIRECTORY_TEMPLATE = "images/{random_id}"
+IMAGE_FILE_NAME_TEMPLATE = "image_{image_counter:09d}.png"
+
```

Comment:
can we please move them to `constants.py`?

Reply:
Sure.